### PR TITLE
Add Filters to Search Form

### PR DESCRIPTION
**Pull request by fatimaburke**
Asana ticket: https://app.asana.com/0/1203289004376659/1204452198889977/f

Screen Shot 2023-05-02 at 4 05 06 PM
Screen Shot 2023-05-02 at 4 05 20 PM

### CODE REVIEW

**Review by littleforest — COMMENTED** [review]
@fatimaburke the filters don't seem to have the same padding that they did on the Figma designs. Just wanted to see if this was correct? Do I need to do anything special to get the assets to work?

Screen Shot 2023-05-02 at 2 24 32 PM

**Comment by littleforest** [comment]
@fatimaburke could you also merge in main and fix merge conflicts?

File: config/tailwind.config.js [hunk 1]
```diff
@@ -7,7 +7,8 @@ module.exports = {
     './public/*.html',
     './app/helpers/**/*.rb',
     './app/javascript/**/*.js',
-    './app/views/**/*.{erb,haml,html,slim}'
+    './app/views/**/*.{erb,haml,html,slim}',
+    './node_modules/flowbite/**/*.js',
```
Comment (littleforest) [inline config/tailwind.config.js:11]: Oh shoot, I totally removed `node_modues` -- @FerPerales is there no way to get flowbite without this?

Comment (FerPerales) [reply config/tailwind.config.js:11]: @littleforest based on the research we did in #20 , this is the recommended way to do it. We were reluctant to introduce the dependency only for closing flash messages, but I think @fatimaburke is considering using more elements

Comment (littleforest) [reply config/tailwind.config.js:11]: @FerPerales but if we are loading flowbite through importmaps, can we access it through there instead?

Comment (FerPerales) [reply config/tailwind.config.js:11]: @littleforest what we are doing through importmaps is adding turbo support, according to the [docs](https://flowbite.com/docs/getting-started/rails/#importmap). We can remove it and see if it works

**Review by littleforest — APPROVED** [review]
🎉 looks great! Would like Fer to review too before merging.

**Pull request merged**